src/components/admin/Settings/PrefixAlerts/Dialog/InitializationError.tsx

```diff
@@ -0,0 +1,70 @@
+import { AlertTitle, Box, Stack, Typography } from '@mui/material';
+import SafeLoadingButton from 'components/SafeLoadingButton';
+import AlertBox from 'components/shared/AlertBox';
+import { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { initializeNotificationSubscriptions } from '../useAlertSubscriptionDialog';
+import useAlertSubscriptionsStore from '../useAlertSubscriptionsStore';
+
+export default function InitializationError() {
```

Comment:
This should be using the standard error component so that is gets the auto handling of checking if the fetch failed and showing that there may be a network issue.